plugins/pdf-extractor/src/plugin.ts

```diff
@@ -0,0 +1,106 @@
+import axios from 'axios'
+import api, { Flatfile } from '@flatfile/api'
+import { FlatfileEvent } from '@flatfile/listener'
+import * as fs from 'fs-extra'
+import * as R from 'remeda'
+
+/**
+ * Plugin config options.
+ *
+ * @property {string} apiKey - `pdftables.com` API key
+ * @property {boolean} debug - show helpful messages useful for debugging (usage intended for development)
+ */
+export interface PluginOptions {
+  readonly apiKey: string
+  readonly debug?: boolean
+}
+
+export const run = async (
+  event: FlatfileEvent,
+  file: Flatfile.File_,
+  buffer: Buffer,
+  opts: PluginOptions
+): Promise<void> => {
+  const { environmentId, spaceId } = event.context
+
+  if (file.ext !== 'pdf' || file.mode !== 'import') {
+    return
+  }
+
+  if (R.isEmpty(opts.apiKey)) {
+    if (opts.debug) {
+      logWarn('Found invalid API key')
```

Comment:
Do we want to show this message even if they don't set debug true? Feels like a pretty silent failure

Reply:
Unless there's a way to use the pdftables api without a key, I kinda feel like this should `logError` and return

Reply:
Agreed. I was too quick to refactor this conditional at the same of the other.